testsuite/performance/README.md

```diff
@@ -47,30 +47,36 @@ Keep reading for more information.
 
 ## Provisioning
 
+### Available provisioners:
+
+- `docker-compose` *Default.* See [`docker-compose/README.md`](docker-compose/README.md) for more details.
+
 ### Provision
 
-Usage: `mvn verify -Pprovision[,cluster] [-D<PARAM>=<VALUE> ...]`. 
+Usage: `mvn verify -Pprovision [-Dprovisioner=<PROVISIONER>] [-D<PARAMETER>=<VALUE>] …`. 
 
-- Single node deployment: `mvn verify -Pprovision`
-- Cluster deployment: `mvn verify -Pprovision,cluster [-Dkeycloak.scale=N]`. Default `N=1`.
+- Single node: `mvn verify -Pprovision`
+- Cluster: `mvn verify -Pprovision,cluster [-Dkeycloak.scale=N] [-Dkeycloak.cpusets="cpuset1 cpuset2 … cpusetM"]`. `N ∈ {1 .. M}`.
```

Comment:
Can't keycloak.scale be clearly derived from keycloak.cpusets (number of elements)?
Having two properties that are linked in this manner is too error-prone. I'd be much surprised that by setting  `keycloak.cpusets` to `4-7 8-11 12-15 16-19`, still the cluster does not use more than 2 instances (because I accidentally forgot to specify `keycloak.scale`).
Please follow [KISS principle](https://en.wikipedia.org/wiki/KISS_principle).

Reply:
I was considering merging those two parameters but in the end I decided not to. I would prefer to keep them separate because 1) the cpusets parameter is only used to derive the maximum scale to which we may want to scale up gradually, and 2) it's docker-specific. In bare-metal lab or cloud the maximum scale can be derived from the list of remote hosts instead of cpusets.

Reply:
Ad 1) That should be taken care of by the runner script e.g. by including more and more cpusets, not left to the user to learn the hard way
Ad 2) In that case these parameters should be described in docker-compose/README.md, not in generic testsuite/performance/README.md

The current way is too prone to human error. Please reconsider your decision.

Reply:
This should be solved, see README.provisioning-parameters.md --> Service Parameters --> Keycloak Server.